xml/System.Net.Http/HttpRequestException.xml

```diff
@@ -141,5 +141,81 @@
         <remarks>To be added.</remarks>
       </Docs>
     </Member>
+    <Member MemberName=".ctor">
+      <MemberSignature Language="C#" Value="public HttpRequestException (string message);" />
```

Comment:
this member duplicates with the member at line 79.

Reply:
You're right, it somehow slipped through, thanks.